tools/depends/native/Makefile

```diff
@@ -22,8 +22,8 @@ ifeq ($(OS),osx)
 endif
 
 ifeq ($(OS),linux)
-  ifeq ($(TARGET_PLATFORM),$(filter $(TARGET_PLATFORM),raspberry-pi gbm))
-    NATIVE += libffi meson ninja python3 setuptools
+  ifeq ($(TARGET_PLATFORM),$(filter $(TARGET_PLATFORM),raspberry-pi aml gbm))
+    NATIVE += meson ninja
```

Comment:
This conflicted please let me know if this is still how it should be @Rechi

Reply:
Why add `aml` (aml windowing), which was dropped in master branch?

Reply:
Why remove `setuptools`, which was added in master branch?

Reply:
Should be resolved with the latest version